docs/xks/developer-guide/report.md

```diff
@@ -0,0 +1,23 @@
+---
+id: reports
+title: Reports
+---
+
+As a XKS 2.0 customer you are getting monthly reports, these reports exist to help to get a quick overview
```

Comment:
Suggestion: "As a XKS 2.0 customer you get monthly reports to get a quick overview of your environment and how your platform is doing."